maplibre/src/coords.rs

```diff
@@ -439,6 +444,7 @@ impl From<Point3<f64>> for WorldCoords {
     }
 }
 
+/// Defines a bounding box on a tiled map with a zoom level and a padding.
```

Comment:
If we add the type `ZoomLevel`:
```suggestion
/// Defines a bounding box on a tiled map with a [`ZoomLevel`] and a padding.
```

Reply:
This should add an in-doc link.